### ISSUE TITLE
Serlib warning

### ISSUE THREAD

**Issue by MRandl** [ejgallego/coq-lsp#800]
**Describe the bug**
On my machine, coq-lsp gives the following warning:
```
Serlib plugin: coq-serapi.serlib.btauto is not available.
Incremental checking for commands in this plugin will be impacted.
```

It happens when attempting to browse files from [github.com/MRandl/verith](https://www.github.com/MRandl/verith)
It does not seem to have an impact on my workflow

**To Reproduce**
This warning was not emitted on my other machine, so it must be environment-dependent
1. clone verith as above
2. make
3. browse Cmod.v
The first import displays the warning 

**Expected behavior**
No warning, as on the other machine

**Screenshots**

**Desktop (please complete the following information):**
 - Arch on linux kernel 6.9.7
 - coq-lsp 0.1.9+8.18
 - coq-lsp extension 0.1.10
 - Codium 1.90.2

**Comment by MRandl** [comment]
I'll gladly add more info on my environment if you need it !

**Comment by ejgallego** [comment]
Thanks @MRandl for tying out LSP and taking the time to submit this issue! 

This is mostly a developer warning, I think time has come to disable it in the 0.2.x. series.

**Comment by MRandl** [comment]
All good then ! Thanks

**Comment by ejgallego** [comment]
Solved by #723
